pages/index.tsx

```diff
@@ -765,3 +765,13 @@ MainPage.getInitialProps = async () => {
 };
 
 export default MainPage;
+
+// helpers
+function getRandomInt(max) {
```

Comment:
You can use types to indicate that `max` is a number, and arrow-function to fit the rest of the code:
```suggestion
const getRandomInt = (max: number) => {
```